src/database/utilities/createRecord.js

```diff
@@ -718,6 +718,38 @@ const createRequisition = (
   return requisition;
 };
 
+const createCustomerRequisition = (
+  database,
+  user,
+  { otherStoreName, program, period, orderType = {} }
+) => {
+  const { REQUISITION_SERIAL_NUMBER } = NUMBER_SEQUENCE_KEYS;
```

Comment:
From what I can tell, customer requisitions share number sequences with supplier requiisitions?

Reply:
My intuition is that they shouldn't... Out of scope for this issue/PR though. Asking about it isn't an immediate problem but isn't clear if it even is one!

Reply:
I created one of each on desktop and they seemed to share, but I'm not gonna pretend to know much about them

Reply:
Yup, I could have been more clear: I checked the number sequence table on desktop, there is only one sequence for requisitions per store. So request/response share the same sequence. Asked @craigdrown, he thought it probably wasn't a problem.